default/etc/e-smith/templates/httpd/proxy-extra/30ProxyPass20RequestHeader

```diff
@@ -0,0 +1,6 @@
+{
+    if ( ($Port == 443) && ($XForwarded eq 'enabled')) {
```

Comment:
We should set the headers also for HTTP port 80
```suggestion
    if ($XForwarded eq 'enabled') {
```

...and select https/http and change 443 with $Port in the lines below.

Please consider also the Apache-based way (if it can work):

```
RequestHeader set "X-Forwarded-Proto" expr=%{REQUEST_SCHEME}
RequestHeader set "X-Forwarded-SSL" expr=%{HTTPS}
```

Reply:
> RequestHeader set "X-Forwarded-Proto" expr=%{REQUEST_SCHEME}
> RequestHeader set "X-Forwarded-SSL" expr=%{HTTPS}

I did never read an example like above on the web, only the example I used can be found in documentation of web application....so :-?

Reply:
The RequestHeader directive is documented here https://httpd.apache.org/docs/current/mod/mod_headers.html#requestheader

Reply:
httpd fails to start with 

```
RequestHeader set "X-Forwarded-Proto" expr=%{REQUEST_SCHEME}
RequestHeader set "X-Forwarded-SSL" expr=%{HTTPS}
```